index.html

```diff
@@ -5,9 +5,12 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
 
-    <link href="//fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
+    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet">
     <link rel="stylesheet" href="styles/normalize.css">
     <link rel="stylesheet" href="styles/main.css">
+    <link rel="icon" type="image/png" href="images/favicon.png">
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
```

Comment:
Might be best to not use jQuery for a simple site. We're always saying at the meetups to use Vanilla JS if possible and then add libs and frameworks if needed. This is a helpful resource: http://youmightnotneedjquery.com/

For the purposes of this PR though,  we can stick with jQuery for the moment and remove it in a future PR.